open_alchemy/cli.py

```diff
@@ -0,0 +1,82 @@
+"""Define the CLI module."""
+import argparse
+import logging
+import pathlib
+
+from open_alchemy import PackageFormat
+from open_alchemy import build_json
+from open_alchemy import build_yaml
+
+# Configure the logger.
+logging.basicConfig(format="%(message)s", level=logging.INFO)
+
+
+def main():
+    """Define the CLI entrypoint."""
+    args = build_application_parser()
+    try:
+        args.func(args)
+    except Exception as exc:  # pylint: disable=broad-except
+        logging.error("Cannot perform the operation: %s.", exc)
+
+
+def build_application_parser():
+    """Build the main parser and subparsers."""
+    # Define the top level parser.
+    parser = argparse.ArgumentParser()
+    subparsers = parser.add_subparsers(title="subcommands", help="subcommand help")
+
+    # Define the parser for the "build" subcommand.
+    build_parser = subparsers.add_parser(
+        "build",
+        description="Build a package with the SQLAlchemy models.",
+        help="build a package",
+    )
+    build_parser.add_argument(
+        "specfile", type=str, help="specify the specification file"
+    )
+    build_parser.add_argument("name", type=str, help="specify the package name")
+    build_parser.add_argument("output", type=str, help="specify the output directory")
+    build_parser.add_argument(
```

Comment:
Is it possible to do both?

Reply:
Addressed ✅

Reply:
Do you think there is any value in calling out that it is optional? I will take your lead on that